scripts/binary/binary-cleanup.js

```diff
@@ -40,8 +40,10 @@ const getDependencyPathsToKeep = async (buildAppDir) => {
   const startingEntryPoints = [
     'packages/server/lib/plugins/child/require_async_child.js',
     'packages/server/lib/plugins/child/register_ts_node.js',
+    'packages/server/node_modules/@cypress/webpack-batteries-included-preprocessor/index.js',
+    'packages/server/node_modules/ts-loader/index.js',
     'packages/rewriter/lib/threads/worker.js',
-    'node_modules/@cypress/webpack-batteries-included-preprocessor/index.js',
+    'npm/webpack-batteries-included-preprocessor/index.js',
```

Comment:
I assume we are replacing these npm paths during the build step?

Reply:
The package process creates an alias at node modules that points here. But the files live here